projects/media-viewer/src/lib/annotations/annotation-set/annotation-create/box-highlight-create.component.ts

```diff
@@ -0,0 +1,133 @@
+import { Component, ElementRef, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import uuid from 'uuid';
+import { Subscription } from 'rxjs';
+import { BoxHighlightCreateService } from './box-highlight-create.service';
+
+@Component({
+  selector: 'mv-box-highlight-create',
+  templateUrl: './box-highlight-create.component.html',
+  styleUrls: ['./box-highlight-create.component.scss']
+})
+export class BoxHighlightCreateComponent {
+
+  @Input() pageHeight: number;
+  @Input() pageWidth: number;
+  @Input() rotate: number;
+  @Input() zoom: number;
+  @Input() container: { top: number, left: number };
+  @Output() highlightCreated = new EventEmitter();
+
+  @ViewChild('boxHighlight') highlight: ElementRef;
+
+  drawStartX = -1;
+  drawStartY = -1;
+  top: string;
+  left: string;
+  height: string;
+  width: string;
+  display: string;
+
+  private subscriptions: Subscription[] = [];
+
+  constructor(private readonly boxHighlightEvents: BoxHighlightCreateService) {}
+
+  ngOnInit(): void {
+    this.subscriptions = [
```

Comment:
I changed it to this because this seems simpler
I would change all the others to this style but didn't want to make this PR even bigger
any components that have more than one subscription should ideally use this